src/writing-policies/rego/02-builtin-support.md

```diff
@@ -0,0 +1,28 @@
+# Builtin support
+
+Building a policy for the `wasm` target is only half of the problem,
+it needs to be executed.
+
+The Open Policy Agent team has a dedicated page you can check in order
+to [find out the built-in support
+level](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions).
+
+Every green check in this table means that those built-ins are
+implemented regardless of the runtime: they are implemented already on
+the policy you have built.
+
+The built-ins marked as `SDK-dependent` are the ones that the host has
+to implement -- in this case, Kubewarden. Open Policy Agent and
+Gatekeeper may use them depending on the needs of the policy. In any
+case, this built-ins are exposed to the policy and any new or existing
+policy could depend on them.
+
+These are the [built-ins implemented up until now in
+Kubewarden](https://github.com/kubewarden/policy-evaluator/issues/56#issue-983937559).
```

Comment:
I would use some contents from the blog post I just wrote, and mix it with something from above:


```
The Rego language provides some [built-in functions](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions)
to perform String, Encoding, Regex, ... operations.
When building a Rego policy into a WebAssembly module, some of these built-in
functions are going to be implemented inside of the Wasm file itself; while others
have to be provided at execution time by the WebAssembly runtime evaluating the module.

The Open Policy Agent team has a dedicated page you can check in order
to [find out the built-in support
level](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions).
Every green check in this table means that those built-ins are
implemented regardless of the runtime: they are implemented already on
the policy you have built.

Kubewarden implements the built-in functions that are not supported natively
by the WebAssembly modules produced by `opa build`. There are still some built-ins that
are not yet provided by us, however, based on the policies we have seen in the open,
the ones we already support should be enough for the majority of Kubernetes users.

[This GitHub issue](https://github.com/kubewarden/policy-evaluator/issues/56)
keeps track of the Rego built-ins we have still to implement. Feel free to
comment over there to prioritize our work.
```